samples/export_model_sample.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_export_model_sample]
+from google.cloud import aiplatform
+
+
+def export_model_sample(
+    gcs_destination_output_uri_prefix: str, project: str, model_id: str
+):
+    client_options = dict(api_endpoint="us-central1-aiplatform.googleapis.com")
+    client = aiplatform.ModelServiceClient(client_options=client_options)
```

Comment:
A comment here is useful to suggest that a user should reuse clients:
```suggestion
    # Initialize client that will be used to create and send requests. This client only needs to be created
    # once, and can be reused for multiple requests.
    client_options = dict(api_endpoint="us-central1-aiplatform.googleapis.com")
    client = aiplatform.ModelServiceClient(client_options=client_options)
```

Reply:
Good call, this comment was added to all AI Platform samples 🙂